mappings/net/minecraft/test/TestContext.mapping

```diff
@@ -259,11 +313,17 @@ CLASS net/minecraft/unmapped/C_ncfucati net/minecraft/test/TestContext
 	METHOD m_xayeypny putAndRemoveRedstoneBlock (Lnet/minecraft/unmapped/C_hynzadkk;J)V
 		ARG 1 pos
 		ARG 2 delay
-	METHOD m_xlrmphsv succeedWhenEntityNotPresent (Lnet/minecraft/unmapped/C_ogavsvbr;III)V
+	METHOD m_xfugxnch createMockSurvivalPlayer ()Lnet/minecraft/unmapped/C_jzrpycqo;
+	METHOD m_xlrmphsv succeedWhenEntityAbsent (Lnet/minecraft/unmapped/C_ogavsvbr;III)V
 		ARG 1 type
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
+	METHOD m_xodhhfcy expectEntitiesAround (Lnet/minecraft/unmapped/C_ogavsvbr;Lnet/minecraft/unmapped/C_hynzadkk;ID)V
+		COMMENT Fails if there is not the given entity {@code count} for the given {@code type} in a box centered around {@code pos}
```

Comment:
Shouldn't `{@code type}` be `{@code testedType}`?

Reply:
Good catch.